src/openscm_units/data/metric_conversions.csv

```diff
@@ -1,92 +1,92 @@
-HFE329mcc2,,,3070
-HFE338mcf2,,,929
-HFE347mcf2,,,854
-HFE356mec3,,,387
-HFE356pcf2,,,719
-HFE356pcf3,,,446
-HFE365mcf3,,,
-HFE374pc2,,,627
-PFPMIE,,10300,9710
-CHCl3,4,,16
-CH2Cl2,9,8.7,9
-CH3Cl,,13,12
-Halon1201,,,376
\ No newline at end of file
+Source,https://www.ghgprotocol.org/sites/default/files/ghgp/Global-Warming-Potential-Values%20%28Feb%2016%202016%29_1.pdf,https://www.ghgprotocol.org/sites/default/files/ghgp/Global-Warming-Potential-Values%20%28Feb%2016%202016%29_1.pdf,https://www.ghgprotocol.org/sites/default/files/ghgp/Global-Warming-Potential-Values%20%28Feb%2016%202016%29_1.pdf,"AR5CCFGWP100 are AR5 100 year GWPs with climate carbon cycle feedbacks: https://www.ipcc.ch/site/assets/uploads/2018/07/WGI_AR5.Chap_.8_SM.pdf, Table 8.SM.16 and https://www.ipcc.ch/site/assets/uploads/2018/02/WG1AR5_Chapter08_FINAL.pdf , Table 8.7 (page 714)."
+OpenSCMMetricName,SARGWP100,AR4GWP100,AR5GWP100,AR5CCFGWP100
+OpenSCM species,,,,
+CH4,21,25,28,34
+N2O,310,298,265,298
+CFC11,3800,4750,4660,5352
+CFC12,8100,10900,10200,11547
+CFC13,,14400,13900,15451
+CFC113,4800,6130,5820,6586
+CFC114,,10000,8590,9615
+CFC115,,7370,7670,8516
+Halon1301,5400,7140,6290,7154
+Halon1211,,1890,1750,2070
+Halon2402,,1640,1470,1734
+Halon1202,,,,280
+CCl4,1400,1400,1730,2019
+CH3Br,,5,2,3
+CH3CCl3,100,146,160,193
+HCFC21,,,148,179
+HCFC22,1500,1810,1760,2106
+HCFC123,90,77,79,96
+HCFC124,470,609,527,635
+HCFC141b,600,725,782,938
+HCFC142b,1800,2310,1980,2345
+HCFC225ca,,122,127,155
+HCFC225cb,,595,525,633
+HFC23,11700,14800,12400,13856
+HFC32,650,675,677,817
+HFC41,150,,116,141
+HFC125,2800,3500,3170,3691
+HFC134,1000,,1120,1337
+HFC134a,1300,1430,1300,1549
+HFC143,300,,328,397
+HFC143a,3800,4470,4800,5508
+HFC152,,,16,20
+HFC152a,140,124,138,167
+HFC161,,,4,4
+HFC227ea,2900,3220,3350,3077
+HFC236cb,,,1210,1438
+HFC236ea,,,1330,1596
+HFC236fa,6300,9810,8060,8998
+HFC245ca,560,,716,863
+HFC245fa,,1030,858,1032
+HFC365mfc,,794,804,966
+HFC4310mee,1300,1640,1650,1952
+SO2F2,,,4090,4732
+SF6,23900,22800,23500,26087
+NF3,,17200,16100,17885
+CF4,6500,7390,6630,7349
+C2F6,9200,12200,11100,12340
+C3F8,7000,8830,8900,9878
+cC4F8,8700,10300,9540,10592
+C4F10,7000,8860,9200,10213
+C5F12,7500,9160,8550,9484
+C6F14,7400,9300,7910,8780
+C7F16,,,7820,8681
+C8F18,,,7620,8456
+C10F18,,,7190,7977
+SF5CF3,,17700,17400,19396
+cC3F6,,,9200,10208
+HFE125,14900,12400,12400,13951
+HFE134,6320,5560,5560,6512
+HFE143a,756,523,523,632
+HCFE235da2,350,491,491,595
+HFE245cb2,708,654,654,790
+HFE245fa2,659,812,812,981
+HFE347mcc3,575,530,530,641
+HFE347pcf2,580,889,889,1072
+HFE356pcc3,110,413,413,500
+HFE449sl,297,421,421,509
+HFE569sf2,59,57,57,69
+HFE4310pccc124,1870,2820,2820,3353
+HFE236ca12,2800,5350,5350,6260
+HFE338pcc13,,1500,2910,3466
+HFE227ea,,,6450,7377
+HFE236ea2,,,1790,2143
+HFE236fa,,,979,1177
+HFE245fa1,,,828,997
+HFE263fb2,,,1,2
+HFE329mcc2,,,3070,3598
+HFE338mcf2,,,929,1118
+HFE347mcf2,,,854,1028
+HFE356mec3,,,387,468
+HFE356pcf2,,,719,867
+HFE356pcf3,,,446,540
+HFE365mcf3,,,,1
```

Comment:
Is there no AR5GWP100 available?

Reply:
It's below 1 and was empty before I started editing the file, so I kept it that way. I didn't add any substances.

Reply:
Ok it seems odd to only have a 'with carbon cycle feedbacks' number but it's not worth losing sleep over.

Reply:
the non-CCF number is "<1" in AR5. We could just put 1 instead of effectively using 0.

Reply:
I'm also happy with that (would probably be my preference to be honest)